docs/source/config.json

```diff
@@ -25,10 +25,10 @@
       "3️⃣ Local composition": "/quickstart/local-composition",
       "4️⃣ Working with subgraphs": "/quickstart/local-subgraphs"
     },
-    "Implementing a Supergraph": {
+    "Building Your Supergraph": {
       "Subgraphs in Apollo Server": "/subgraphs",
       "Other subgraph libraries": "/supported-subgraphs",
-      "Graph router/gateway": "/gateway"
+      "The graph router/gateway": "/gateway"
```

Comment:
should we update the link to `/router` as well?

Reply:
ah that's fair, sure I'll make that change